libs/client/ui/pages/src/lib/profile/profile.tsx

```diff
@@ -274,6 +282,33 @@ export const Profile: React.FC<ProfileProps> = inject(userStore.storeKey)(
                 Ändern
               </IonButton>
             </IonRow>
+            <IonRow className="profile__content__preferences">
+              <IonItem>
+                <IonLabel>Präferenzen</IonLabel>
+                <IonSelect
+                  multiple={true}
+                  value={selecPreference}
+                  selectedText=" "
+                  onIonChange={(e) => setSelecPreference(e.detail.value)}
+                >
+                  <IonSelectOption value={Difficulty.ADVANCED}>
+                    Fortgeschritten
+                  </IonSelectOption>
+                  <IonSelectOption value={Difficulty.BEGINNER}>
+                    Anfänger
+                  </IonSelectOption>
+                  <IonSelectOption value={Difficulty.GAME}>
+                    Spiele
+                  </IonSelectOption>
+                </IonSelect>
+              </IonItem>
+            </IonRow>
+            <IonRow className="profile__content__rankcheck">
+              <IonItem>
+                <IonCheckbox slot="start"></IonCheckbox>
```

Comment:
Warum hosch ned da custom component in `libs\client\ui\components\src\lib\checkbox\checkbox.tsx` verwendet? Denn hätts o mitm styles besser hinghaut.